Makefile

```diff
@@ -21,7 +21,7 @@ BIN := ark
 PKG := github.com/heptio/ark
 
 # Where to push the docker image.
-REGISTRY ?= gcr.io/heptio-images
+REGISTRY ?= vorstella
```

Comment:
I believe this change will have to be reverted; others don't have access to the Vorstella registry :)

Reply:
oops, will correct.